src/modules/common/components/input-dropdown/input-dropdown.styles.less

```diff
@@ -73,6 +73,14 @@
 
 .InputDropdown__icon {
   align-self: center;
```

Comment:
this should be flex-end so the arrow is aligned

Reply:
made the correction.